### ISSUE TITLE
Only Liking 27 post at a time

### ISSUE THREAD

**Issue by Emilbackhausen** [InstaPy/InstaPy#1456]
im not sure why! looked around in the issues box but cant find anyone with this problem.... im locked to only liking 27 posts, cant see why... cant find a solution... please help (im using the quickstart.py)

**Comment by ghost** [comment]
Same problem

**Comment by Emilbackhausen** [comment]
@brokleen Do you have any error messages?

**Comment by Emilbackhausen** [comment]
@brokleen Check out this, it solved the problem for me! 
https://github.com/timgrossmann/InstaPy/issues/1365#issuecomment-366526517

**Comment by kevXZY** [comment]
same issue here ...

**Comment by GabrielleV** [comment]
I had the same issue, updated to the last version, and now it's only linking 12 posts!

**Comment by uluQulu** [comment]
hello @GabrielleV 
have you checked [this](https://github.com/timgrossmann/InstaPy/issues/1365#issuecomment-366526517)?

**Comment by lucanello** [comment]
@uluQulu your fix should be added to the repository. Since you already have a PR open, should I make a PR with your changes? Or maybe @timgrossmann can manually change it.

**Comment by uluQulu** [comment]
sure, @lucanello but I think it's better to wait people test it a bit and after a good feedback I will generalize new `get_links()` function to work with other similarly handled features and maybe then merge with main

**Comment by lucanello** [comment]
Okay sure! It's working brilliantly for me. I wasn't able to work further on my add-ons but I've seen the logs and all accounts liked the specified amount of images. Well done!

**Comment by GabrielleV** [comment]
Thank you so much @uluQulu my life is better because of you (I don't even exaggerate!)

**Comment by stainless33** [comment]
When trying to use this fix I get the following error:
```
Traceback (most recent call last):
  File "quickstart.py", line 1, in <module>
    from instapy import InstaPy
  File "/root/InstaPy/instapy/__init__.py", line 2, in <module>
    from .instapy import InstaPy
  File "/root/InstaPy/instapy/instapy.py", line 24, in <module>
    from .like_util import get_links_from_feed
ImportError: cannot import name 'get_links_from_feed'
```

**Comment by uluQulu** [comment]
hi @stainless33 
there seems to be an indentation mistake
I can check if you share `like_util.py`, just [gist](https://gist.github.com/) it

**Comment by stainless33** [comment]
@uluQulu I probably overlooked something as per usual. [Here](https://gist.github.com/stainless33/7862e082d86f5994b55ed9160fa4aa8b) it is.

**Comment by uluQulu** [comment]
@stainless33 
you've mistakenly replaced `get_links_from_feed()`
here is corrected [file](https://gist.github.com/uluQulu/35ac02e9ef58e4a6263346e13aa0e7f2)

**Comment by stainless33** [comment]
@uluQulu Ah, crap, that was stupid - but now I see what I did incorrectly. Thank you for taking the time to have a look at it and for enhancing the overall utility of likability for stainless steel. ;)

**Comment by timgrossmann** [comment]
Should be fixed by #1844
